ts/Core/Utilities.ts

```diff
@@ -1850,16 +1860,17 @@ function getNestedProperty(path: string, parent: unknown): unknown {
  * @param {boolean} [toInt=true]
  *        Parse to integer.
  *
- * @return {number|string}
- *         The numeric value.
+ * @return {number|string|undefined}
+ *         The style value.
  */
 function getStyle(
     el: HTMLDOMElement,
     prop: string,
     toInt?: boolean
-): (number|string) {
+): (number|string|undefined) {
+    const customGetStyle: typeof getStyle = ((H as any).getStyle || getStyle);
 
-    var style;
+    let style: (number|string|undefined) = (toInt ? NaN : void 0);
```

Comment:
I can't imagine a case where we actually want to use `NaN`. Would it be a lot of overhead if we return `number|undefined` and check validity in the implementations?

Reply:
`parseInt` is also returning NaN, so I see it as the natural value for failed integer parsing.